src/utils/keychain.js

```diff
@@ -48,38 +48,41 @@ export type KeyChainData = {
   pin?: ?string,
 };
 
-export const handleCatch = (accountAddress: ?string, error: ?any) => {
-  reportErrorLog('Exception caught on keychain: ', error);
-  const colors = getThemeColors();
-  const buttons = [];
-  buttons.push({
-    text: t('error.failedKeychain.exitButtonText'),
-    onPress: () => RNExitApp.exitApp(),
-  });
-  accountAddress = accountAddress ?? etherspotService?.getAccountAddress(CHAIN.ETHEREUM);
-  if (accountAddress) {
+export const handleCatch = (accountAddress: ?string, error: ?any[]) => {
+  const msg = error ? error[0]?.Error?.msg : '';
+  if (msg && !/cancel/gi.exec(msg)) {
+    reportErrorLog('Exception caught on keychain: ', error);
```

Comment:
Please move the error reporting outside the `if` statement, we want to report all errors even if they are just actions that are cancelled.

Reply:
okay